src/github.com/getlantern/autoupdate/mobile.go

```diff
@@ -0,0 +1,159 @@
+package autoupdate
+
+import (
+	"bytes"
+	"compress/bzip2"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/blang/semver"
+	"github.com/getlantern/flashlight/config"
+	"github.com/getlantern/flashlight/proxied"
+	"github.com/getlantern/go-update"
+)
+
+var (
+	updateServer = config.DefaultUpdateServerURL
+	//updateServer = "https://update-stage.getlantern.org/update"
```

Comment:
Uncomment this line to use the autoupdate staging server during testing otherwise you will get an error about Android not being supported